client/public/src/components/Navbar.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react'
+import { Link, withRouter } from 'react-router-dom'
+
+class Navbar extends Component {
+    logOut(e) {
+        e.preventDefault()
+        localStorage.removeItem('usertoken')
```

Comment:
You shouldn't store a token in the local storage. I see that you use it to populate user info in the `Profile` component. This should be changed with a request to backend.